poke-line.el

```diff
@@ -137,29 +151,29 @@ Minimum of 3 units are required for poke-line."
            (pokemon-string (propertize "|||" 'display (create-image (poke-line-get-pokemon-image) 'png nil :ascent 'center)))
            (background-string "")
            (buffer (current-buffer)))
-           (dotimes (number elements)
-             (setq element-string
-               (concat element-string
-                 (poke-line-add-scroll-handler
-                   (if png-support
-                       (propertize "|" 'display (create-image (poke-line-get-element-image) 'png nil :ascent 'center))
-                     "|")
-                   (/ (float number) poke-line-bar-length) buffer))))
-             (dotimes (number backgrounds)
-               (setq background-string
-                 (concat background-string
-                   (poke-line-add-scroll-handler
-                     (if png-support
-                         (propertize "-" 'display (create-image poke-line-background-image 'png nil :ascent 'center))
-                       "-")
-                     (/ (float (+ elements poke-line-size number)) poke-line-bar-length) buffer))))
+      (dotimes (number elements)
+        (setq element-string
+              (concat element-string
+                      (poke-line-add-scroll-handler
+                       (if png-support
+                           (propertize "|" 'display (create-image (poke-line-get-element-image) 'png nil :ascent 'center))
+                         "|")
+                       (/ (float number) poke-line-bar-length) buffer))))
+      (dotimes (number backgrounds)
+        (setq background-string
+              (concat background-string
+                      (poke-line-add-scroll-handler
+                       (if png-support
+                           (propertize "-" 'display (create-image poke-line-background-image 'png nil :ascent 'center))
+                         "-")
+                       (/ (float (+ elements poke-line-size number)) poke-line-bar-length) buffer))))
       ;; Compute Poke Cat string.
       (propertize
-        (concat
-          pokemon-string
-          element-string
-          background-string)
-        'help-echo poke-line-modeline-help-string))))
+       (concat
+        pokemon-string
+        element-string
+        background-string)
+       'help-echo poke-line-modeline-help-string))))
```

Comment:
I think these should be indented 2 spaces like the rest of the file

Reply:
Nope, this is how Emacs indents it, so it's objectively correct. I highly recommend using `aggressive-indent-mode` when developing elisp to avoid discrepancies creeping in.

Reply:
Will check it out

Reply:
Fun related fact: we got pretty close to adding strict indentation checks to `package-lint`, and only didn't do so because user-defined macros can be tagged with indentation hints, so you can't know unambiguously how a file should be indented without loading it.